src/Microsoft.OData.Edm/Validation/EdmError.cs

```diff
@@ -25,12 +25,31 @@ public EdmError(EdmLocation errorLocation, EdmErrorCode errorCode, string errorM
             this.ErrorLocation = errorLocation;
             this.ErrorCode = errorCode;
             this.ErrorMessage = errorMessage;
+            this.ErrorLevel = "error";

```

Comment:
@wachugamaina Is this really necessary? Would there be any issue with leaving `ErrorLevel` unassigned (i.e. with a value of `null`)? `error` does not seem like a valid error level to me so I think it's better to leave it with a null value if its not available

Reply:
Resolved. Added None as default.